program/rust/src/processor/upd_price.rs

```diff
@@ -126,6 +131,18 @@ pub fn upd_price(
         }
     }
 
+    #[cfg(test)]
```

Comment:
this doesn't need `cfg(test)` right? like it only triggers once the account is resized, and right now they haven't been resized?